src/main/java/com/gojek/beast/worker/BqQueueWorker.java

```diff
@@ -57,6 +57,8 @@ private Status pushToSink(Records poll) {
         Status status;
         try {
             status = sink.push(poll);
+            statsClient.gauge("batch.records.size," + statsClient.getBqTags(), poll.getSize());
+            statsClient.gauge("batch.records.count," + statsClient.getBqTags(), poll.getRecords().size());
```

Comment:
Let us rename the metric worker.queue.bq.processing to worker.queue.bq.processing.time to be explicit.